src/main/java/edu/cornell/kfs/concur/rest/application/ConcurApplication.java

```diff
@@ -0,0 +1,31 @@
+package edu.cornell.kfs.concur.rest.application;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.core.Application;
+
+import edu.cornell.kfs.concur.service.impl.ConcurWebServiceImpl;
+
+@ApplicationPath("/concur/v1.0")
```

Comment:
I removed this code for now as it is not related to testconnection. When I add it back with the next user story I will change it to use v1.